src/MongoDB.Bson/BsonUtils.cs

```diff
@@ -110,6 +110,18 @@ public static char ToHexChar(int value)
             return (char)(value + (value < 10 ? '0' : 'a' - 10));
         }
 
+        /// <summary>
+        /// Converts a value to a hex character.
+        /// </summary>
+        /// <param name="value">The value (assumed to be between 0 and 15).</param>
+        /// <returns>The hex character.</returns>
+#pragma warning disable CS3001
```

Comment:
There are a lot of suppressing like this in this PR. Otherwise, all public members/methods which contain `uint` will generate the following warning:

    Type is not CLS-comliant.

Reply:
This method is not needed if you accept my other suggestions.